### DISCUSSION TITLE
Need you help in making decision on ZincSearch Logo

### DISCUSSION THREAD

**Discussion by hengfeiyang** [zinclabs/zinc#148] [Show and tell]
Hello good folks here. Need you help in making decision on ZincSearch Logo. Please upvote one of these.

**Comment by hengfeiyang** [comment]
![logo1](https://user-images.githubusercontent.com/1628250/167899478-4a59329a-7941-41ef-9dd1-ac09fe4b3ec9.png)

**Reply by B0bbyD0llar** [reply]
I think this is more beautiful than the other. However, I also see a certain risk of confusion with the Subway logo.

**Reply by prabhatsharma** [reply]
Haha, I too see that now. We will change the colors...

**Reply by jergoo** [reply]
I like the style of this one as it clearly stands for Z, the other one looks like an Atlassian product style to me from the first look. 
But I'm not sure about the color of this one, you see, the logo of Elastic is made of colorful blocks, and the primary color is also yellow and green(not the same).😁

**Reply by GeekRu1** [reply]
I prefer it, more impressed.Sharp "Z"

**Comment by hengfeiyang** [comment]
![image (2)](https://user-images.githubusercontent.com/1628250/167899530-a63f7c37-bf8e-4aea-9d0b-58dc19e470ff.png)

**Comment by naemrashid** [comment]
Voting for 1. Also, just a thought, as zinc is going to be a lightweight and lightning-fast search engine, I think this should be reflected in the logo too :) so my suggestion is to use flash reference https://dynamic.brandcrowd.com/asset/logo/812be7a1-101b-46da-aafc-092e227e7fd8/logo-search-grid-1x?v=637849793816830000. So logo no. 1 with Z as in image.
